app/controllers/application_controller.rb

```diff
@@ -3,6 +3,7 @@
 class ApplicationController < ActionController::Base
   before_action :set_paper_trail_whodunnit
   include ApplicationHelper
+  include Pagy::Backend
```

Comment:
This is fine, though it may make sense to include this only on the resources when necessary.